tests/award.js

```diff
@@ -377,6 +377,6 @@ module.exports = {
                         done();
                     })
             })
-            .end();
+            //.end();
     }
```

Comment:
there's an issue with this test that might need to be fixed, I think it's due to some async issue where the test is terminating before the async stuff has completed. Might have to look into this a bit more.